modules/swagger-jaxrs/src/main/java/com/wordnik/swagger/jaxrs/DefaultParameterExtension.java

```diff
@@ -102,12 +97,23 @@ else if(annotation instanceof FormParam) {
     return parameters;
   }
 
-  public boolean shouldIgnoreClass(Class<?> cls) {
-    boolean output = false;
-    if(cls.getName().startsWith("javax.ws.rs"))
-      output = true;
-    else
-      output = false;
-    return output;
+  @Override
+  protected boolean shouldIgnoreClass(Class<?> cls) {
+    return cls.getName().startsWith("javax.ws.rs.");
```

Comment:
Changed `rs` to `rs.` to skip `javax.ws.rs.*` classes only.
Removed state variable to get shorter body.